.circleci/config.yml

```diff
@@ -28,14 +28,11 @@ cache-key: &cache-key
 commands:
   with_npm_token:
```

Comment:
As it is now only used within the release step, can we remove it and use the command directly?